src/main/java/de/torui/coflsky/CoflSkyCommand.java

```diff
@@ -95,8 +95,13 @@ public void processCommand(ICommandSender sender, String[] args) throws CommandE
 				//			+ "g º87,000 -> 13,999 ºg[BUY]\\\",\\\"onClick\\\":\\\"/viewauction f7d7295ca72f43e9876bf6da7424000c\\\",\\\"hover\\\":\\\"\\\"}\"}"), sender.getCommandSenderEntity());
 				//WSCommandHandler.HandleCommand(new Command(CommandType.PlaySound, "{\"name\":\"random.orb\",\"pitch\":0.5}"), sender.getCommandSenderEntity());
 					break;	
-				case "callback":
-					CallbackCommand(args);
+					case "callback":
+						CallbackCommand(args);
+						break;
+				case "dev":
+					CoflSky.Wrapper.initializeNewSocket("ws://localhost:8009/modsocket");
+					Config.BaseUrl = "http://localhost:5005";
+					System.out.println("entered dev mode");
```

Comment:
Should there be any indication that this requires a full restart to be able to connecto to the online servers again?

Reply:
made it toggle-able and write in chat where its connected to